WordPressComStatsiOS/WordPressComStatsiOS/UI/WPStatsGraphBarCell.h

```diff
@@ -9,6 +9,7 @@
 @property (nonatomic, strong) NSArray *categoryBars;
 
 @property (nonatomic, copy) NSString *barName;
+@property CGFloat barNameFontSize;
```

Comment:
Mind adding `(nonatomic)` ?. By default, properties are `atomic`, and this way we'd avoid the extra locking overhead.

[Ref. here!](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html)